src/overlays/actors/ovl_Bg_Mori_Hineri/z_bg_mori_hineri.h

```diff
@@ -6,6 +6,9 @@
 
 struct BgMoriHineri;
 
+// Due to a unique access pattern, this param cannot use the generic "PARAMS_GET_U" macros
+#define TWISTED_HALLWAY_GET_PARAM_15(thisx) (((thisx)->params & (NBITS_TO_MASK(1) << (15))) >> ((15) - (1)))
```

Comment:
I hate how overwritten this is.

Reply:
Its written in the same form as the generic macros. The point is to easily let you see the bit width and position.

Reply:
Its also 1 of 2 cases where a custom actor-specific actor macro is used. Imo we can figure out these edge cases another time, like when the actor is documented

Reply:
Personally I don't find that the generic macro easy to follow logically in it's raw form. Probably more important though is that the macro isn't a proper params getter; the bit gets extracted but then you have a secondary operation "baked in" of the value being shifted into the bit 1 spot instead of bit 0 like the majority of params getters. This is an important detail that is being obfuscated by the macro, and makes it hard to follow the subsequent logic. 

In this unusual instance, I feel `PARAMS_GET_NOSHIFT` would be the more appropriate way to extract the value. The right shift needs to be exposed to the user.

Reply:
Fwiw I agree with mxz here. PARAMS_GET_NOSHIFT seems better so you can explicitly show the unique shift (probably the same for en_ishi as well which seems to have the same structure). However also agree we could wait until the actors are documented.

Reply:
can either of you write out how it would be used with NOSHIFT? I would be willing to change it now

Reply:
`#define TWISTED_HALLWAY_GET_PARAM_15(thisx) (PARAMS_GET_NOSHIFT((thisx)->params, 15, 1) >> 14)`
Oh wait en_ishi is different. Don't think NOSHIFT works. My bad.

Reply:
en_ishi does work like `PARAMS_GET_NOSHIFT(PARAMS_GET_NOMASK((thisx)->params, 12 - 2), 2, 4)` but they may be a bit too much.

Reply:
I would do something more like: 

    #define TWISTED_HALLWAY_GET_TYPE_NOSHIFT(x) PARAMS_GET_NOSHIFT(x, 15, 1)
    ...
    this->dyna.actor.params = TWISTED_HALLWAY_GET_TYPE_NOSHIFT(this->dyna.actor.params) >> 14;

Reply:
I suggest just reverting this (to main), and we can indeed figure it out (or not) later, but not let it stop the whole thing